tools/lib/provision.py

```diff
@@ -144,85 +152,103 @@
+if vendor == "debian" and os_version in [] or vendor == "ubuntu" and os_version in []:
     # For platforms without a pgroonga release, we need to build it
     # from source.
     BUILD_PGROONGA_FROM_SOURCE = True
-    SYSTEM_DEPENDENCIES = UBUNTU_COMMON_APT_DEPENDENCIES + [
-        pkg.format(POSTGRES_VERSION) for pkg in [
-            "postgresql-{0}",
-            # Dependency for building pgroonga from source
-            "postgresql-server-dev-{0}",
-            "libgroonga-dev",
-            "libmsgpack-dev",
-            "clang-9",
-            "llvm-9-dev",
+    SYSTEM_DEPENDENCIES = (
+        UBUNTU_COMMON_APT_DEPENDENCIES
+        + [
+            pkg.format(POSTGRES_VERSION)
+            for pkg in [
+                "postgresql-{0}",
+                # Dependency for building pgroonga from source
+                "postgresql-server-dev-{0}",
+                "libgroonga-dev",
+                "libmsgpack-dev",
+                "clang-9",
+                "llvm-9-dev",
+            ]
         ]
-    ] + VENV_DEPENDENCIES
+        + VENV_DEPENDENCIES
+    )
 elif "debian" in os_families():
-    SYSTEM_DEPENDENCIES = UBUNTU_COMMON_APT_DEPENDENCIES + [
-        pkg.format(POSTGRES_VERSION) for pkg in [
-            "postgresql-{0}",
-            "postgresql-{0}-pgroonga",
-        ]
-    ] + VENV_DEPENDENCIES
+    SYSTEM_DEPENDENCIES = (
+        UBUNTU_COMMON_APT_DEPENDENCIES
+        + [pkg.format(POSTGRES_VERSION) for pkg in ["postgresql-{0}", "postgresql-{0}-pgroonga"]]
+        + VENV_DEPENDENCIES
+    )
 elif "rhel" in os_families():
-    SYSTEM_DEPENDENCIES = COMMON_YUM_DEPENDENCIES + [
-        pkg.format(POSTGRES_VERSION) for pkg in [
-            "postgresql{0}-server",
-            "postgresql{0}",
-            "postgresql{0}-devel",
-            "postgresql{0}-pgroonga",
+    SYSTEM_DEPENDENCIES = (
+        COMMON_YUM_DEPENDENCIES
+        + [
+            pkg.format(POSTGRES_VERSION)
+            for pkg in [
+                "postgresql{0}-server",
+                "postgresql{0}",
+                "postgresql{0}-devel",
+                "postgresql{0}-pgroonga",
+            ]
         ]
-    ] + VENV_DEPENDENCIES
+        + VENV_DEPENDENCIES
+    )
 elif "fedora" in os_families():
-    SYSTEM_DEPENDENCIES = COMMON_YUM_DEPENDENCIES + [
-        pkg.format(POSTGRES_VERSION) for pkg in [
-            "postgresql{0}-server",
-            "postgresql{0}",
-            "postgresql{0}-devel",
-            # Needed to build pgroonga from source
-            "groonga-devel",
-            "msgpack-devel",
+    SYSTEM_DEPENDENCIES = (
+        COMMON_YUM_DEPENDENCIES
+        + [
+            pkg.format(POSTGRES_VERSION)
+            for pkg in [
+                "postgresql{0}-server",
+                "postgresql{0}",
+                "postgresql{0}-devel",
+                # Needed to build pgroonga from source
+                "groonga-devel",
+                "msgpack-devel",
+            ]
```

Comment:
This block might benefit from naming a `postgres_dependencies` object rather than adding it without a name to something else, given how Black adds a bunch of nesting when doing so.